pkg/agent/baker.go

```diff
@@ -984,9 +984,16 @@ func getContainerServiceFuncMap(config *datamodel.NodeBootstrappingConfiguration
 		"InsertIMDSRestrictionRuleToMangleTable": func() bool {
 			return config.InsertIMDSRestrictionRuleToMangleTable
 		},
+		"IsArc": func() bool {
+			return isArc(config)
+		},
```

Comment:
What the purpose of it?
I think RP can't provision arc nodes. Is it for testing purposes?